src/services/jwt.service.js

```diff
@@ -0,0 +1,50 @@
+import jwt from 'jsonwebtoken';
+
+const generateAccessToken = (user) => {
+  return jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '10m',
+  });
+};
+
+const generateRefreshToken = (user) => {
+  return jwt.sign(user, process.env.JWT_REFRESH_KEY);
```

Comment:
The refresh token should also have an expiration time set for security reasons. Consider adding an 'expiresIn' option to the jwt.sign method.